File: frontend/src/types/audit.ts
export interface AuditAttachment {
  file_name: string;
  file_url:  string;
}

export interface Audit {
  id:                string;
  srg_id:            string;
  ot_factura:        string;
  observations:      string;
  concesionaria:     string;
  auditor_id:        string;
  additional_emails: string[];
  attachments:       AuditAttachment[];
  created_at:        string;
  updated_at:        string;
}

File: backend/src/domain/entities/audit.py
from dataclasses import dataclass, field
from typing import Optional
from uuid import UUID, uuid4

from .base import BaseEntity


@dataclass
class AuditAttachment:
    file_name: str
    file_url: str


@dataclass
class AuditEntity(BaseEntity):
    srg_id: UUID = field(default_factory=uuid4)
    ot_factura: str = ""
    observations: str = ""
    concesionaria: str = ""
    auditor_id: Optional[UUID] = None
    additional_emails: list[str] = field(default_factory=list)
    attachments: list[AuditAttachment] = field(default_factory=list)
